lib/gc/Transforms/GPU/Pipeline.cpp

```diff
@@ -35,8 +35,7 @@ void populateGPUPipeline(OpPassManager &pm,
     pm.addNestedPass<func::FuncOp>(createAddContextArg());
   }
 
-  pm.addNestedPass<func::FuncOp>(createIterativeTilingAndFusion());
```

Comment:
`createIterativeTilingAndFusion` could be kept after `createGpuTilingAndFusion()` which may perform some further fusion

Reply:
Do you have an example? I need to test it, but I haven't seen any real cases.

Reply:
You could refer to these serveral PRs https://github.com/llvm/llvm-project/pull/111955, https://github.com/llvm/llvm-project/pull/94190, https://github.com/llvm/llvm-project/pull/88712

Reply:
Checked the first 2 and I don't see any difference with and without createIterativeTilingAndFusion

Reply:
ok, we could remove it now and see whether we will meet the case that requires a more powerful fusion.